icu4c/source/common/ubrk.cpp

```diff
@@ -174,6 +174,22 @@ ubrk_safeClone(
     return (UBreakIterator *)newBI;
 }
 
+U_CAPI UBreakIterator * U_EXPORT2
+ubrk_clone(const UBreakIterator *bi, UErrorCode *status) {
+    if (U_FAILURE(*status)) {
+        return nullptr;
+    }
+    if (bi == nullptr) {
```

Comment:
By the usual ICU coding guidelines, we don't normally check service object pointers for null. And, indeed, in most of the other ubrk_whatever() functions, we don't.

Making this change would also require removing some test code.

Reply:
Done.